TheInternetTestSuit/AuthorizationTests.cs

```diff
@@ -0,0 +1,56 @@
+global using NUnit.Framework;
+using Authentication;
+using Welcome;
+
+namespace TheInternetTestSuit
+{
+    [TestFixture]
+    public class Authorization : BaseTest
+    {
+        private LoginPage loginPage;
+        private LogoutPage logoutPage;
+        private WelcomePage welcomePage;
+        private SecureAreaPage secureAreaPage;
+
+        [SetUp]
+        public void TestSetup()
+        {
+            Driver.Navigate().GoToUrl(MainUrl);
+            loginPage = new LoginPage(Driver);
+            welcomePage = new WelcomePage(Driver);
+            secureAreaPage = new SecureAreaPage(Driver);
+            logoutPage = new LogoutPage(Driver);
```

Comment:
Maybe we can initialize our pages only ones in OneTimeSetUp here?

Reply:
I encountered problems, can I leave this way?